code/drasil-example/pdcontroller/lib/Drasil/PDController/Body.hs

```diff
@@ -90,7 +92,7 @@ mkSRS
                  ShowDerivation,
                Constraints EmptyS inputsUC]],
 
-     ReqrmntSec $ ReqsProg [FReqsSub EmptyS [], NonFReqsSub], LCsSec,
+     ReqrmntSec $ ReqsProg [FReqsSub inputValuesDescription [], NonFReqsSub], LCsSec,
```

Comment:
I'm sure this is a good change, but it is going to lead to changes in `stable`. So it is usually best to do such changes in isolation (even if it means five 1-line change PRs) instead of being bundled in a larger PR.